src/engraving/rendering/dev/tdraw.cpp

```diff
@@ -2068,3 +2071,100 @@ void TDraw::draw(const Rest* item, Painter* painter)
     painter->setPen(item->curColor());
     item->drawSymbol(item->sym(), painter);
 }
+
+//! NOTE Can be removed later (should be only single mode)
```

Comment:
I doubt whether this will be the case, because the shadow note _does_ need to know about the context (staff type, score font, etc.).